pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -1476,6 +1479,20 @@ type ContainerRuntime struct {
 	ProviderConfig *runtime.RawExtension `json:"providerConfig,omitempty" protobuf:"bytes,2,opt,name=providerConfig"`
 }
 
+// WorkersSettings contains settings for all workers.
+type WorkersSettings struct {
+	// SSHAccess containes setting to enable/disable SSH access to the worker nodes.
+	// +optional
+	SSHAccess *SSHAccess `json:"sshAccess,omitempty" protobuf:"bytes,1,opt,name=sshAccess"`
+}
+
+// SSHAccess containes setting to enable/disable SSH access to the worker nodes.
```

Comment:
I am not quite happy with the single setting description of this struct. This is subject to change in the future and I am not sure that this comment will be updated accordingly if this happens. Let's just provide a more general comment here. Something like: 
```suggestion
// SSHAccess contains settings regarding ssh access to the worker nodes.
```

Mind that this text occurs on several places and they should also be changed accordingly.